modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/UserCardTag.java

```diff
@@ -0,0 +1,584 @@
+		_inputValue = inputValue;
+	}
+
+	public void setLabels(List<LabelItem> labels) {
+		_labels = labels;
+	}
+
+	public void setName(String name) {
+		_name = name;
+	}
+
+	public void setSelectable(Boolean selectable) {
+		_selectable = selectable;
+	}
+
+	public void setSelected(Boolean selected) {
+		_selected = selected;
+	}
+
+	/**
+	 * @deprecated As of Cavanaugh (7.4.x), with no direct replacement
+	 */
+	@Deprecated
+	public void setSpritemap(String spritemap) {
+		_spritemap = spritemap;
+	}
+
+	public void setSubtitle(String subtitle) {
+		_subtitle = subtitle;
+	}
+
+	public void setUserCard(UserCard userCard) {
+		_userCard = userCard;
+	}
+
+	public void setUserColorClass(String userColorClass) {
+		_userColorClass = userColorClass;
+	}
+
+	@Override
+	protected void cleanUp() {
+		super.cleanUp();
+
+		_actionDropdownItems = null;
+		_disabled = null;
+		_groupName = null;
+		_href = null;
+		_icon = null;
+		_imageAlt = null;
+		_imageSrc = null;
+		_inputName = null;
+		_inputValue = null;
+		_labels = null;
+		_name = null;
+		_selectable = null;
+		_selected = null;
+		_spritemap = null;
+		_subtitle = null;
+		_userCard = null;
+		_userColorClass = null;
+	}
+
+	@Override
+	protected String getHydratedModuleName() {
+		return "frontend-taglib-clay/UserCard";
+	}
+
+	@Override
+	protected Map<String, Object> prepareProps(Map<String, Object> props) {
+		props.put("actions", getActionDropdownItems());
+		props.put("description", getSubtitle());
+		props.put("disabled", getDisabled());
+		props.put("href", getHref());
+		props.put("id", getId());
+		props.put("inputName", getInputName());
+		props.put("inputValue", getInputValue());
+		props.put("userImageAlt", getImageAlt());
+		props.put("userImageSrc", getImageSrc());
+		props.put("userSymbol", getIcon());
+		props.put("labels", getLabels());
+		props.put("name", getName());
+		props.put("selectable", getSelectable());
+		props.put("selected", getSelected());
+		props.put("spritemap", getSpritemap());
+		props.put("userDisplayType", getUserColorClass());
+
+		return super.prepareProps(props);
+	}
+
+	@Override
+	protected String processCssClasses(Set<String> cssClasses) {
+		Boolean selectable = getSelectable();
+
+		if ((selectable == null) || !selectable) {
+			cssClasses.add("card");
+		}
+
+		cssClasses.add("card-type-asset");
+		cssClasses.add("form-check");
+		cssClasses.add("form-check-card");
+		cssClasses.add("form-check-top-left");
+		cssClasses.add("user-card");
+
+		return super.processCssClasses(cssClasses);
+	}
+
+	@Override
+	protected int processStartTag() throws Exception {
+		super.processStartTag();
+
+		JspWriter jspWriter = pageContext.getOut();
+
+		jspWriter.write("<div class=\"card\">");
+		jspWriter.write("<div class=\"aspect-ratio card-item-first\">");
+
+		Boolean disabled = getDisabled();
+		Boolean selectable = getSelectable();
+
+		if ((selectable != null) && selectable) {
+			jspWriter.write("<div class=\"custom-control custom-checkbox\">");
```

Comment:
@carloslancha @jbalsas  Is there a way to set content of a tag in Java? We can do this in [React](https://github.com/liferay/clay/blob/master/packages/clay-card/src/CardWithHorizontal.tsx#L128) and [Soy](https://github.com/liferay/clay/blob/2.x/packages/clay-card/src/ClayHorizontalCard.soy#L63-L67), but I cannot find a way to do it in Java. This would make it possible to use `CheckboxTag` here.

Reply:
The only way I found was something like to create a new `pageContext`, write on it and pass it to the `setBody` or `setContentBody` or sth like that. And I couldn't make it work properly.

So the best solution IMHO is what I did for the `StickerTag`, just add a `content` attribute that places the value where children should be renderer.

Reply:
Yup, that's the way to do it... the expectation is that we won't be touching this often, so we can stick with this for now and only reevaluate if we find ourselves updating these often and causing inconsistencies.

Reply:
> So the best solution IMHO is what I did for the StickerTag, just add a content attribute that places the value where children should be renderer.

This would be good if it was a simple sting, but the content is complex logic with lots of `write`s. 

I'll stick with plain HTML as well (`<div class=\"custom-control custom-checkbox\">` etc.)

Reply:
You can use `StringBundler` if you need it https://github.com/liferay-frontend/liferay-portal/pull/434/files#diff-7df0e8ed3761415125433a5b6417c3f72c6b95449afa355c819a7cfccf796daeR441